index.html

```diff
@@ -2096,7 +2096,12 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           "rfc2119" title="MUST">MUST</em> include an entry in the array
           associated with the name <code>additionalResponses</code> that
           includes a value for the name <code>schema</code>.</span> 
-          </p></section>
+          </p><p>
+          A <a>Thing Description</a> of a WoT producer may has multiple forms entries with, e.g., different 
```

Comment:
nit: may have

Reply:
done

Reply:
Still looks the same, don't forget you need to fix these comments in 2 places (copy-paste) - and I see you fixed it one place, not here. You can amend the previous commit I guess.

Reply:
sorry, I forgot the render script